Dockerfile

```diff
@@ -1,8 +1,21 @@
 FROM ubuntu:xenial-20181005
 
-LABEL maintainer="sameer@damagehead.com"
+ARG BUILD_DATE
+ARG VCS_REF
+ARG VERSION
```

Comment:
Specify default version. This will also people to build it  without specifing the version.

```
VERSION=11.4.5
```

Reply:
DONE